### DISCUSSION TITLE
Gameficação da indicação

### DISCUSSION THREAD

**Discussion by FelipeQueiroz** [diego3g/rsxp-2023#5] [Ideas]
Acharia interessante gameficar o sistema de indicações como exemplo:

Fazer uma barra de progresso de acordo com as pessoas que convida e de 5 em 5 desbloquear beneficios pré evento e dentro do evento. 

E talvez a cada premiação liberar uma figurinha para compatilhar na comunidade. Acho que tem muito espaço pra essa ideia e acredito que pode ser uma boa forma de interação com o app.

Acho que a partir daqui podemos criar uma discussão bem construtiva para fazer uma feature bacana :)

**Comment by eletroswing** [comment]
A gameficação com essas figurinhas pode ser bastante interativa também! por exemplo, a pessoa poder compartilhar uma barra de quanto do álbum ela completou, ou trocar isso por pontos pra trocar por descontos ou algo assim!

**Comment by ruymon** [comment]
@FelipeQueiroz @eletroswing  Deem uma olhada nessas ideias aqui #8 #3. Acho que são similares a esta aqui. :)

**Reply by FelipeQueiroz** [reply]
Acredito que a ideia deste tópico seria em relação apenas ao programa de indicação, mas acharia super interessante ela ser integrada com os stickers como citado nesses tópicos :)

**Comment by sdavi38** [comment]
A ideia é válida para melhor interação com app e criar valores para quem deseja baixar o app

**Comment by FelipeQueiroz** [comment]
Estava procurando algumas referencias e a ideia é ser só uma implementação básica, como essa progress bar que tirei do dribble: 

<img width="503" alt="Screenshot 2023-03-28 at 21 21 43" src="https://user-images.githubusercontent.com/28742231/228395301-69409448-69b7-4b9b-9e0f-74910bd810e5.png">

<img width="880" alt="Screenshot 2023-03-28 at 21 22 47" src="https://user-images.githubusercontent.com/28742231/228395413-ec7921c1-d2f5-4afd-beb6-eb6011aa6f26.png">

A ideia é implemenetar uma simples progress bar de acordo com a quantidade de amigos chamados, dai acredito que ela pode se encaixar nessa página de "Convidar" feita pelo @diego3g 

oque acham?

**Comment by felipecepluki** [comment]
Acho uma boa @FelipeQueiroz, estou disposto a contribuir com essa feature também

**Comment by felipecepluki** [comment]
Como posso saber o que vocês estão fazendo para não contribuir com código repetido ?
